config.yaml

```diff
@@ -0,0 +1,11 @@
+options:
+  cuda-version:
+    type: string
+    default: "8.0.61"
+    description: |
+      The version of CUDA to be intalled.
+  cuda-subversion:
```

Comment:
There is only this place https://github.com/juju-solutions/layer-nvidia-cuda/blob/master/reactive/cuda.sh#L285 where CUDA_VERSION is used without CUDA_SUBVERSION. Should we use one config param and split on the last "-" to get the subversion?